lib/oai/provider/response/list_metadata_formats_decorator.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module OAI::Provider::Response::ListMetadataFormatsDecorator
+  def record_supports(record, prefix)
+    prefix == 'oai_dc' or
```

Comment:
Is there a reason for using `or` instead of `||`?

In Ruby, `||` is a short-circuit.  The first operand that evaluates `true` ends the check of all subsequent operands.  Whereas with `or` you keep checking each operand.

(Similarly `&&` and `and`)

Reply:
This one I'm not sure about. Since I am overriding the oai gem, I [kept it the same as what was there](https://github.com/code4lib/ruby-oai/blob/master/lib/oai/provider/response/list_metadata_formats.rb#L32-L36).